warehouse/testhelper/setup.go

```diff
@@ -376,13 +385,20 @@ func VerifyingTablesEventCount(t testing.TB, wareHouseTest *WareHouseTest) {
 		t.Logf("Verifying tables event count for sqlStatement: %s", sqlStatement)
 
 		require.Contains(t, wareHouseTest.EventsCountMap, table)
-		tableCount := wareHouseTest.EventsCountMap[table]
+		var tableCount int
+		if table == "google_sheet" {
```

Comment:
Why not pass directly wh_google_sheet instead of proxying it to using the condition `table == "google_sheet"`

Reply:
This is because the table created is called google_sheets in the warehouse.